pkg/ccl/changefeedccl/changefeed_test.go

```diff
@@ -1172,6 +1177,59 @@ func TestChangefeedAuthorization(t *testing.T) {
 	t.Run(`enterprise`, enterpriseTest(testFn))
 }
 
+func TestChangefeedFailOnRBRChange(t *testing.T) {
+	defer leaktest.AfterTest(t)()
+	defer log.Scope(t).Close(t)
+
+	skip.UnderRace(t)
+	skip.UnderShort(t)
```

Comment:
does the test take long time to run?

Reply:
Not the fastest but not the slowest either:

```
--- PASS: TestChangefeedFailOnRBRChange (1.32s)
    --- PASS: TestChangefeedFailOnRBRChange/sinkless (0.48s)
        --- PASS: TestChangefeedFailOnRBRChange/sinkless/regional_by_row (0.28s)
    --- PASS: TestChangefeedFailOnRBRChange/enterprise (0.82s)
        --- PASS: TestChangefeedFailOnRBRChange/enterprise/regional_by_row (0.53s)
```